webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py

```diff
@@ -56,6 +56,29 @@ async def test_prompt_type(
     }
 
 
+async def test_prompt_default_value(
+    bidi_session, inline, new_tab, subscribe_events, wait_for_event
+):
+    await subscribe_events(events=[USER_PROMPT_OPENED_EVENT])
+    on_entry = wait_for_event(USER_PROMPT_OPENED_EVENT)
+
+    text = "test"
+    default = "default"
+
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"],
+        url=inline(f"<script>window.prompt('{text}', '{default}')</script>"),
+    )
+
+    event = await on_entry
+
+    assert event == {
+        "context": new_tab["context"],
+        "type": prompt_type,
```

Comment:
`prompt_type` variable is not defined in this test, maybe that causes the failures.